README.md

```diff
@@ -31,19 +40,36 @@ Developers include:
  * Richard Hornung (hornung1@llnl.gov)
  * Randolph Settgast (settgast1@llnl.gov)
 
+Please see the `BLT Contributors Page <https://github.com/LLNL/BLT/graphs/contributors>`_ for the full list of project contributors.
+
+Projects using this library
+---------------------------
+
+ * `Ascent <https://github.com/Alpine-DAV/ascent>`_: A flyweight in-situ visualization and analysis runtime for multi-physics HPC simulations
+ * `Axom <https://github.com/LLNL/axom>`_: Software infrastructure for the development of multi-physics applications and computational tools
+ * `CHAI <https://github.com/LLNL/CHAI>`_: Copy-hiding array abstraction to automatically migrate data between memory spaces
+ * `Conduit <https://github.com/LLNL/conduit>`_: Simplified data exchange for HPC simulations
+ * `RAJA <https://github.com/LLNL/raja>`_: Performance portability layer for HPC
+ * `Umpire <https://github.com/LLNL/Umpire>`_: Application-focused API for memory management on NUMA and GPU architectures
+ * `VTK-h <https://github.com/Alpine-DAV/vtk-h>`_: Scientific visualization algorithms for emerging processor architectures
```

Comment:
Looks like github's markdown is not happy with these links.  

- [ ] Fix this.